src/java.base/share/classes/java/lang/runtime/ObjectMethods.java

```diff
@@ -396,15 +396,16 @@ private static MethodHandle makeToString(MethodHandles.Lookup lookup,
      *                     if invoked by a condy
      * @throws IllegalArgumentException if the bootstrap arguments are invalid
      *                                  or inconsistent
-     * @throws NullPointerException if any argument but {@code lookup} is {@code null},
-     *                              in the case of the {@code getters} argument, its
+     * @throws NullPointerException if any argument is {@code null}, in the case
+     *                              of the {@code getters} argument, its
```

Comment:
OK. The second part about the contents of `getters` is a bit clunky, but was previously approved.